run_docker.sh

```diff
@@ -0,0 +1,14 @@
+# Build image
+docker build -t rwschubert/kalman-laika .
```

Comment:
Is it intended that this file contains "**rwschubert**/kalman-laika", on line 2 and line 13? Just wondering if that's an artifact from your workspace or something.

Reply:
It is just the tag of the Docker image that is chosen when the Docker image is built.
By specifying it with `-t` you could set another tag.